content/en/tracing/security/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Security and Fine-tuning
+kind: documentation
+description: "Configure the Datadog Tracer or Agent to modify or discard spans for security or fine-tuning purposes."
+aliases:
+    - /security/tracing
+    - /tracing/guide/security
+further_reading:
+    - link: 'tracing/security/agent_span_modifications'
+      tag: 'Documentation'
+      text: 'Configure the Datadog Agent to modify or discard spans.'
+    - link: 'tracing/tracer_span_modification'
+      tag: 'Documentation'
+      text: 'Configure the Tracer to modify or discard spans.'
+---
+## Overview
+
+The Datadog Agent and all tracing libraries have several options available to modify or discard spans.
```

Comment:
This page should open with why people might want to do this, so they know if they're in the right place. Something like: "The performance data and traces that you're collecting with Datadog can contain sensitive information that you want to filter out, obfuscate, scrub, filter, modify, or just not collect."